packages/node/src/login/oidc/oidcHandlers/ClientCredentialsOidcHandler.spec.ts

```diff
@@ -263,9 +309,7 @@ describe("handle", () => {
           clientType: "static",
         },
       })
-    ).rejects.toThrow(
-      /Invalid response from Solid Identity Provider \[.+\]: \{.+\} is missing 'id_token'/
-    );
+    ).rejects.toThrow();
```

Comment:
shouldn't we test the error thrown here?

Reply:
Good point, added in e5795b4f630aecbf8add18a1e2d74e41dd4bf81a.